### ISSUE TITLE
Separate jellyfin server management URL and invite form URL

### ISSUE THREAD

**Issue by Spiritreader** [hrfee/jellyfin-accounts#16]
Currently you can only set the jellyfin server URL once which is used both for the management access for creating accounts as well as for the invite form.

This way I cannot remove the "allow remote access" checkbox for the admin account handling the logins.

Ideally the two URLs can be separated. This way the invite link works fine and the management access should work without account remote access enabled.

Is this scenario possible?

```
[jellyfin]
; it is reccommended to create a limited admin account for this program.
username = admin
password = $PASS
; server will also be used in the invite form, so make sure it's publicly accessible.
server = http://$URL 
client = jf-accounts
version = 0.1
device = jf-accounts
device_id = jf-accounts-0.1
```

**Comment by DER31K** [comment]
You can just use  a reverse proxy, to make any url link to anything you want... I already do this.

**Comment by Spiritreader** [comment]
I'm afraid I don't quite understand what you mean with that.

I am using a reverse proxy for jellyfin. However the URL showing up in the "success" popup is the one I set in the [jellyfin] configuration instead of the reverse proxied jellyfin url.

https://github.com/hrfee/jellyfin-accounts/blob/cb07cbb5bfe56adb20ea7af4263c37d729099f9c/jellyfin_accounts/web.py#L63

Changing this to 
```
jfLink=config['jellyfin']['public_server'],
```

and modifying the config file to say
```
[jellyfin]
; it is reccommended to create a limited admin account for this program.
username = admin
password = $PASS
; server will also be used in the invite form, so make sure it's publicly accessible.
server = http://$INTERNAL_URL
public_server =  https://EXTERNAL_URL
client = jf-accounts
version = 0.1
device = jf-accounts
device_id = jf-accounts-0.1
```

does what I described.

jf-account can authenticate by using a local IP where the admin acct is whitelisted on, and the success message displays the public server url.

What is your intended way to achieve this?
